src/components/MyObservations/SimpleUploadBannerContainer.js

```diff
@@ -176,7 +180,7 @@ const SimpleUploadBannerContainer = ( {
 
   // hide when there are less than two observations to upload
   // but make sure completed status is displayed when uploads are finished
-  if ( numOfUserObservations < 2 && progress === 0 ) {
+  if ( isDefaultMode && numOfUserObservations < 2 && progress === 0 ) {
```

Comment:
From what I can see in the designs, I think this check here should not be for default mode but for currentUser.
 In these screens here in the column for all users the number shown is 1:
 https://www.figma.com/design/MChpvx4ZrKEVVwsWKt4lkI/iNaturalist-Mobile-UI-Design?node-id=23655-68746&t=pMuk9C20LY8bFrMb-4
 https://www.figma.com/design/MChpvx4ZrKEVVwsWKt4lkI/iNaturalist-Mobile-UI-Design?node-id=23655-73876&t=pMuk9C20LY8bFrMb-4
 Only for logged out users the number 1 is not shown:
 https://www.figma.com/design/MChpvx4ZrKEVVwsWKt4lkI/iNaturalist-Mobile-UI-Design?node-id=23655-66337&t=pMuk9C20LY8bFrMb-4